prow/git/git.go

```diff
@@ -147,7 +155,7 @@ func (c *Client) Clone(repo string) (*Repo, error) {
 		if err := os.MkdirAll(filepath.Dir(cache), os.ModePerm); err != nil && !os.IsExist(err) {
 			return nil, err
 		}
-		remote := fmt.Sprintf("%s/%s", base, repo)
+		remote := fmt.Sprintf("%s/%s", base.Path, repo)
```

Comment:
initial clone is failing because of this, fix is here: https://github.com/kubernetes/test-infra/pull/11214